src/bindings/python/src/pyopenvino/core/common.cpp

```diff
@@ -134,9 +257,18 @@ py::array as_contiguous(py::array& array, ov::element::Type type) {
 }
 
 py::array array_from_tensor(ov::Tensor&& t, bool is_shared) {
+    // Special case for string data type.
+    if (t.get_element_type() == ov::element::string) {
+        PyErr_WarnEx(PyExc_RuntimeWarning,
```

Comment:
may be add a check if is_shared then throw warning?

Reply:
Actually, the goal is to redirect users to `bytes_str` and `string_str` to use it directly -- the behavior differs for strings and it's better to use them accordingly in different scenarios.